scripts/check-lockfiles.mjs

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2023, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import path from 'node:path'
```

Comment:
We always grumble about the scripts being stuck using `require()`, so this I made this as a `.mjs` file so it can use imports.